index.js

```diff
@@ -63,7 +63,11 @@ module.exports = function(app) {
         res.sendFile(viewsDirectory + '/front.html');
     });
 
-    app.use(function (err, req, res) {
-        if (err) return res.status(err.code || 500).send(err);
+    app.use(function (err, req, res, next) {
+        if (err) {
```

Comment:
Based on the [doc](http://expressjs.com/en/guide/error-handling.html), a handler with 4 parameters can only be an error handler, so there is no need to test for `err`.